lib/inspec/plugins.rb

```diff
@@ -6,4 +6,37 @@ module Inspec
   module Plugins
     autoload :Resource, 'inspec/plugins/resource'
   end
+
+  class PluginCtl
+    attr_reader :registry
+
+    def initialize(home = nil)
+      @home = home || File.join(Dir.home, '.inspec', 'plugins')
+      @paths = Dir[File.join(@home, '**{,/*/**}', '*.gemspec')]
+               .map { |x| File.dirname(x) }
+               .map { |x| Dir[File.join(x, 'lib', 'inspec-*.rb')] }
+               .flatten
+      # map paths to names
+      @registry = Hash[@paths.map { |x|
+        [File.basename(x, '.rb'), x]
+      }]
+    end
+
+    def list
```

Comment:
Nitpicking, but how about `extend Forwardable` and `def_delegator :@registry, :keys, :list`?